project/webapp/templates/user/settings.html

```diff
@@ -119,15 +119,13 @@
 <script type="text/javascript" src="{% static "js/settings.js" %}"></script>
 <script type="text/javascript" src="{% static "js/utils/locate.js" %}"></script>
 <script type="text/javascript">
-    var googleMapsApiKey = "{{ google_map_api_key }}";
-
     location_data = {
         lng: {{lng}},
         lat: {{lat}}
     }
 
     var mapModel = new cw.Map({coordinates: location_data});
-    var mapView = new cw.MapView({model: mapModel, googleMapsApiKey: googleMapsApiKey});
+    var mapView = new cw.MapView({model: mapModel});
```

Comment:
I believe we should also remove the `mapView` and `location_data` since we are moving away from storing user locations at all. This code may have been accidentally left over from PR #875

Reply:
@brylie I've made this change here but I see that they are used at a few other places in some js. I'm not really confident of removing them though. I guess we should start having a test suite so that these changes can be made more confidently